### ISSUE TITLE
question about the true label

### ISSUE THREAD

**Issue by yujeong-star** [huanglianghua/siamfc-pytorch#37]
In the siamfc paper, the true label is composed of 1 and -1, but I think the true label here uses 1 and 0.
I'm wondering whether this can cause any performance drop or not.
Thanks in advance :)

**Comment by huanglianghua** [comment]
Binary cross entropy loss requires labels to be between 0~1, so these labels make sense as probabilities.
[1, -1] is traditionally used in, e.g., SVMs, as coefficients. The SiamFC paper may use this formulation for convenience.
Use 0, 1 will not drop the performance.
